test-butler-app-core/src/main/java/com/linkedin/android/testbutler/AlwaysFinishActivitiesChanger.java

```diff
@@ -30,43 +27,48 @@
 
     private static final String TAG = AlwaysFinishActivitiesChanger.class.getSimpleName();
 
+    private final SettingsAccessor settings;
+
     private boolean originalAlwaysFinishActivitiesMode;
 
+    AlwaysFinishActivitiesChanger(SettingsAccessor settings) {
```

Comment:
Can you make sure we have either `@NonNull` or `@Nullable` on any method parameters of method return types?